addons/account/models/account_tax.py

```diff
@@ -785,8 +785,18 @@ def _get_generation_dict_from_tax_line(self, line_vals):
             'analytic_account_id': line_vals['analytic_account'].id if tax.analytic else False,
         }
 
+    def _get_tax_repartition_lines_sum_factor(self, is_refund):
+        self.ensure_one()
+        factor = 0.0
+        repartition_line_ids = 'refund_repartition_line_ids' if is_refund else 'invoice_repartition_line_ids'
+        for line in self[repartition_line_ids]:
+            if line.repartition_type == 'tax':
+                factor += line['factor']
+        return factor
+
     def _compute_taxes(self, base_lines, tax_lines=None,
-                       handle_price_include=True, include_caba_tags=False):
+                       handle_price_include=True, include_caba_tags=False,
+                       quick_edit_total_amount=None):
```

Comment:
Why are you not giving it a `base_lines` with the right amount instead?

Reply:
You don't need any change in this file.
Control the rounding errors from `_onchange_quick_edit_total_amount` directly